apps/api/src/shared/service/level.service.ts

```diff
@@ -15,4 +15,8 @@ export class LevelService {
   async findAll(): Promise<Level[]> {
     return await this.levelModel.find().exec();
   }
+
+  async findOne(id: string): Promise<Level> {
+    return await this.levelModel.findById(id).populate('logos');
```

Comment:
how this populate works? cause I see there is a call to the logosService on the find level method on the controller

Reply:
Yeah, I was doing some tests and this was left behind. It's definitely not needed. I'll get rid of it.